iphone/Classes/TiUITabGroup.m

```diff
@@ -391,6 +391,40 @@ - (void)setActiveTabBackgroundImage_:(id)value
   controller.tabBar.selectionIndicatorImage = [self loadImage:value];
 }
 
+- (void)setTabsTransparent_:(id)value
+{
+  UITabBar *tabBar = [controller tabBar];
+
+  if ([TiUtils boolValue:value def:NO]) {
```

Comment:
You're already checking the value inside the iOS 13 specific block. I think this guard needs to be removed here...